app/adapters/stripe-subscription.js

```diff
@@ -0,0 +1,5 @@
+import ApplicationAdapter from './application';
+
+export default ApplicationAdapter.extend({
```

Comment:
What are your thoughts on just creating a `StripeAdapter` that extends from the application adapter and sets `coalesceFindRequests` to false, and then all of these adapter could extend from that?